spec/requests/api/blueprints_spec.rb

```diff
@@ -41,10 +41,14 @@
 
     it "will show a blueprint's bundle if requested" do
       blueprint = FactoryGirl.create(:blueprint, :name => "Test Blueprint")
-      service_catalog = FactoryGirl.create(:service_template_catalog)
-      service_dialog = FactoryGirl.create(:dialog_with_tab_and_group_and_field)
-      service_templates = FactoryGirl.create_list(:service_template, 2)
-      blueprint.create_bundle(service_templates, service_dialog, service_catalog)
+      options =
+        {
+          "service_templates" => FactoryGirl.create_list(:service_template, 2),
+          "service_catalog"   => FactoryGirl.create(:service_template_catalog),
+          "service_dialog"    => FactoryGirl.create(:dialog_with_tab_and_group_and_field)
+        }
+      blueprint.create_bundle(options)
```

Comment:
@imtayadeway I updated the other comment. This one I will leave as is to avoid line too long violation.